pkg/profilecreator/profilecreator.go

```diff
@@ -312,6 +307,34 @@ func (ghwHandler GHWHandler) GetReservedAndIsolatedCPUs(reservedCPUCount int, sp
 
 }
 
+// getCPUsSequentially returns Reserved and Isolated CPUs sequentially
+func (ghwHandler GHWHandler) getCPUsSequentially(reservedCPUCount int, topologyInfoNodes []*topology.Node) (string, string, error) {
+	totalCPUSet := cpuset.NewBuilder()
+	reservedCPUSet := cpuset.NewBuilder()
+	max := reservedCPUCount
+	htEnabled, err := ghwHandler.isHyperthreadingEnabled()
+	if err != nil {
+		return "", "", fmt.Errorf("Error determining if Hyperthreading is enabled or not: %v", err)
+	}
```

Comment:
this block should be moved in the calling site and both helper functions should gain a `htEnabled bool` argument

Reply:
Sure, can do that, no problem!